### PR TITLE
Adds Qualifications to Interview Search Page and Scheduling Process

### PR DESCRIPTION
**Pull request by meagancheese**
Screenshot 2020-07-31 at 1 36 19 PM
Screenshot 2020-07-31 at 1 36 29 PM
Screenshot 2020-07-31 at 1 36 40 PM
Screenshot 2020-07-31 at 1 36 49 PM
Screenshot 2020-07-31 at 1 36 58 PM

### CODE REVIEW

**Comment by meagancheese** [comment]
Still one bug I'm working on.

File: src/main/java/com/google/sps/servlets/LoadInterviewsServlet.java [hunk 1]
```diff
@@ -144,6 +157,10 @@ private static ZoneOffset convertIntToOffset(int timezoneOffsetMinutes) {
       userId = String.format("%d", userEmail.hashCode());
     }
     interviewers.remove(userId);
+    // We need to check that the interviewers are qualified to give an interview for the specified
+    // position
```
Comment (apluscs) [inline src/main/java/com/google/sps/servlets/LoadInterviewsServlet.java]: Nit: period at end of comment

File: src/main/java/com/google/sps/data/InterviewPostRequest.java [hunk 2]
```diff
@@ -18,18 +18,30 @@
...
+
+  public void fixNotSpecifiedFields() {
+    if (company.equals("<Not specified>")) {
+      company = "";
+    }
+    if (job.equals("<Not specified>")) {
+      job = "";
+    }
+    return;
   }
```
Comment (apluscs) [inline src/main/java/com/google/sps/data/InterviewPostRequest.java]: Personally l feel like this should all be done in the Javascript. Backend should deal with "clean" data
@acmcarther @jalexanderqed

**Review by apluscs — APPROVED** [review]

**Review by GrantComm — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/LoadInterviewsServlet.java (hunk 1 above)
Comment (meagancheese) [reply src/main/java/com/google/sps/servlets/LoadInterviewsServlet.java]: Done

File: src/main/java/com/google/sps/data/InterviewPostRequest.java (hunk 2 above)
Comment (meagancheese) [reply src/main/java/com/google/sps/data/InterviewPostRequest.java]: Jack asked for this earlier, I'll take the time to do that now.

Comment (meagancheese) [reply src/main/java/com/google/sps/data/InterviewPostRequest.java]: I'm not sure how to do this with the modal. Will ask at sync.

Comment (meagancheese) [reply src/main/java/com/google/sps/data/InterviewPostRequest.java]: Done

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**